src/presentational-components/shared/toolbar.js

```diff
@@ -154,7 +160,12 @@ export const filterConfigBuilder = (
                       name: value,
                       orderBy: sortBy,
                     })
-                  );
+                  ).then((data) => {
+                    if (textFilterRef) {
```

Comment:
Do we need to have the `if` in here as well? Conditional chaining should take care of it.

Reply:
(I want to kill conditional chaining :smiling_imp: )